proxy/proxymanager_test.go

```diff
@@ -349,3 +349,130 @@ func TestProxyManager_StripProfileSlug(t *testing.T) {
 	assert.Equal(t, http.StatusOK, w.Code)
 	assert.Contains(t, w.Body.String(), "ok")
 }
+
+// Test issue #61 `Listing the current list of models and the loaded model.`
+func TestProxyManager_RunningEndpoint(t *testing.T) {
+	// Define a helper struct to parse the JSON response
+	type RunningResponse struct {
```

Comment:
this struct can be shared between `listRunningProcessesHandler` and the tests. Then `gin.H` does not need to be used.